Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/CapsuleComponentMode.cpp

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "CapsuleComponentMode.h"
```

Comment:
Apologies this was my fault giving @greerdv bad advice - I have done this in the past, and for private .h/.cpp pairs I do think it's okay (and it might actually be slightly faster) but I realize this goes against the general guidelines so `<>` is probably the best option to default to - I'm just an old curmudgeon stuck in my ways 😅 🙈

Reply:
Switched to <> style everywhere in this PR. Also found one redundant include that could be removed.